lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -86,17 +86,31 @@
 from ansible.module_utils.gcp_utils import GcpSession, navigate_hash, GcpRequestException
 from ansible.plugins.inventory import BaseInventoryPlugin, Constructable, Cacheable, to_safe_group_name
 import json
+import os
 
 
 # The mappings give an array of keys to get from the filter name to the value
 # returned by boto3's GCE describe_instances method.
 class GcpMockModule(object):
     def __init__(self, params):
         self.params = params
+        self._set_fallbacks()
 
     def fail_json(self, *args, **kwargs):
         raise AnsibleError(kwargs['msg'])
 
+    def _set_fallbacks(self):
+        self._set_fallback('project', 'GCP_PROJECT')
```

Comment:
ENV vars should be supported by adding them to DOCUMENTATION.  The `config` mechanism supports this natively, see other inventory plugins for examples.

Reply:
Fixed! I pulled in the `gcp` doc fragment.